app/representers/concerns/uri_methods.rb

```diff
@@ -49,4 +94,18 @@ def prx_model_uri_part_to_string(part)
       end
     end
   end
+
+  def method_missing(method_name, *args, &block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.